### PR TITLE
chore(deps): update dependencies and settings for CKAN 2.11 compatibility

### PR DESCRIPTION
**Pull request by Markus92**
New DCAT extension is required

New harvest extension is necessary. There is no release yet (latest release is June '23), so tagged the latest current commit in master branch

`recline_view` plugin is unsupported in 2.11, so replaced with `datatables_view`

Due to being debian-based, using apt-get to install dependencies. supervisor is done in base image, we need to add postgres client for loading the translations table. cron is not installed by default and I cannot test atm how well it'll work.

Some more changes might be necessary, but a first smoketest with this seems to work.

### CODE REVIEW

File: ckan/Dockerfile [hunk 1]
```diff
@@ -4,26 +4,24 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-only
 
-FROM ckan/ckan-base:2.10.5
+FROM ckan/ckan-base:2.11
```
Comment (brunopacheco1) [inline ckan/Dockerfile:7]: important: the same comments done on `Dockerfile.dev` are applied for this file.

**Review by brunopacheco1 — CHANGES_REQUESTED** [review]
important: let's wait a bit more to integrate these changes, as they impact the whole system, and we have soon enough a release to do.

File: ckan/Dockerfile.dev [hunk 2]
```diff
@@ -4,23 +4,21 @@
...
 RUN pip3 install -e git+https://github.com/CivityNL/ckanext-scheming.git@release-3.0.0-civity#egg=ckanext-scheming[requirements]    
 
 RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-fairdatapoint.git@v1.3.0#egg=ckanext-fairdatapoint && \
     pip3 install -r ${APP_DIR}/src/ckanext-fairdatapoint/requirements.txt
 
-# Install Supervisor and cron
-RUN apk update \
-    && apk add --no-cache supervisor cronie \
-    && rm -rf /var/cache/apk/*
+# # Install cron and postgres client
```
[the remark this answers is not in the document]
Comment (Markus92) [reply ckan/Dockerfile.dev:20]: Postgres client is necessary for the upload_vocabulary script, which uses the psql command. CKAN works without it.

File: ckan/Dockerfile.dev [hunk 3]
```diff
@@ -53,7 +51,7 @@ RUN for d in $APP_DIR/patches/*; do \
     fi ; \
     done
 
-ENV CKAN__PLUGINS="envvars image_view text_view recline_view scheming_datasets scheming_organizations gdi_userportal dcat harvest ckan_harvester dcat_rdf_harvester dcat_json_harvester dcat_json_interface fairdatapointharvester activity"
+ENV CKAN__PLUGINS="envvars image_view datatables_view text_view scheming_datasets scheming_organizations gdi_userportal dcat harvest ckan_harvester dcat_rdf_harvester dcat_json_harvester dcat_json_interface fairdatapointharvester activity"
```
[the remark this answers is not in the document]
Comment (Markus92) [reply ckan/Dockerfile.dev:54]: We don't need any of those. The dcat_json harvesters/interface can also be removed.

**Comment by Markus92** [comment]
Fixed the issues. I'd still advise waiting for merging this in until ckanext-dcat 2.0 scheming and changes are prepared.

**Comment by brunopacheco1** [comment]
Hey @Markus92 , just to let you know the new harvest release is out there and our ckan-docker has it installed, and also latest dcat extension, fair-datapoint and scheming changes.

Could you please also pay attention to the impacts on UI? We have made quite a lot of theme changes, that may not work correctly anymore.

File: ckan/Dockerfile.dev (hunk 3 above)
[the remark this answers is not in the document]
Comment (brunopacheco1) [reply ckan/Dockerfile.dev]: If we are sure we don't use none of these, and there is not dependency on them from other extensions, indeed we could simply remove them all.

**Comment by hcvdwerf** [comment]
> @hcvdwerf didn't you already take a look at this?

not yet, but require only settings the versions correct and do some tests

**Pull request merged**